src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -59,7 +59,10 @@ internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyRe
 
         internal override ImmutableArray<NamedTypeSymbol> GetAdditionalTopLevelTypes()
         {
-            return _additionalTypes;
+            var builder = ArrayBuilder<NamedTypeSymbol>.GetInstance();
+            builder.AddRange(_additionalTypes);
+            builder.AddIfNotNull(SourceModule.GetNullableAttribute());
```

Comment:
>builder.AddIfNotNull(SourceModule.GetNullableAttribute()); [](start = 12, length = 58)

Is this temporary code? Perhaps mark it with PROTOTYPE comment.

Reply:
Added `// PROTOTYPE` comment.